.github/DISCUSSION_TEMPLATE/ideas.yml

```diff
@@ -0,0 +1,43 @@
+title: "(packageName) Summary of feature or enhancement"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for wishing to improve packages that we host in this repository.
+
+        Before you fill out the template here, please make sure that the title contains a very small summary of the the new enhancement or feature, that the title starts with `(packageName)`, and that can be give an idea what the discussion is about.
+        **NOTE: Replace `packageName` with the name of the package**
+  - type: checkboxes
+    attributes:
+      label: Before starting, ensure that you have done the following.
+      options:
+        - label: I have verified that there is no other issue or discussion covering the feature or enhancement
```

Comment:
Similar to the next option, should this one specify open issue or discussions? If we're not specifying open issues, should we be explicit about open and closed? I know for myself, I rarely search for closed issues (because by default the search is just for open ones).

Reply:
In theory, it should be both open and closed issues.
If an issue was closed, there could be a reason for it.

Reply:
That's fair.